src/main/java/com/sigopt/model/Experiment.java

```diff
@@ -162,58 +166,9 @@ public PropertyResource(String prefix, String name, Class<T> klass) {
         }
     }
 
-    public static class Subresource<T extends APIObject> extends BoundObject {
-        String name;
-        Class<T> klass;
-
+    public static class Subresource<T extends APIObject> extends com.sigopt.model.Subresource<T> {
         public Subresource(String prefix, String name, Class<T> klass) {
```

Comment:
AFAIK you can just omit this, and leasve the class body empty as `{}`

Reply:
Complains, I think because of the super(prefix) in the parent.

Reply:
got it. makes sense